app/components/Layout/Footer.jsx

```diff
@@ -52,20 +47,15 @@ class Footer extends React.Component {
     }
 
     checkNewVersionAvailable(){
-        let request = oneoff();
-
-        request.open("GET", "https://api.github.com/repos/bitshares/bitshares-ui/releases/latest", true);
-        request.send(null);
-        request.addEventListener("load", function(){
-            try {
-                let json = JSON.parse(request.responseText);
-                let oldVersion = String(json.tag_name);
-                let newVersion = String(APP_VERSION);
-                if(oldVersion !== newVersion){
-                    this.setState({newVersion});
-                }
-            } catch(e){}
-        }.bind(this), false);
+        fetch("https://api.github.com/repos/bitshares/bitshares-ui/releases/latest").then((res)=>{
```

Comment:
I meant to only perform the fetch itself if it's actually needed with the `if (__ELECTRON__)` check.